roles/kubernetes/preinstall/tasks/dhclient-hooks.yml

```diff
@@ -1,8 +1,8 @@
 ---
-- name: Configure dhclient to prepend nameservers and supersede search/domain
+- name: Configure dhclient to supersede search/domain/nameservers
   blockinfile:
     block: |-
-      {% for item in [ supersede_domain, supersede_search, prepend_nameserver ] -%}
+      {% for item in [ supersede_domain, supersede_search, supersede_nameserver ] -%}
```

Comment:
I don't understand the loop here instead of {{ supersede_domain }}\n{{supersede_search}}...etc